wire/gen_peer_wire_csv

```diff
@@ -102,6 +102,25 @@ accept_channel2,204,htlc_basepoint,33
 accept_channel2,237,first_per_commitment_point,33
 accept_channel2,270,opening_tlv_len,2
 accept_channel2,272,opening_tlv+,opening_tlv_len
+funding_compose,58
+funding_compose,0,temporary_channel_id,32
+funding_compose,32,num_inputs,2
+funding_compose,34,input_info,$num_inputs
+funding_compose,34+$,num_outputs,2
+funding_compose,36+$,output_info,$num_outputs
```

Comment:
Maybe insist that subtypes are defined *before* use, and then you can just recognize the  typename here, no need for $?

Reply:
That makes the RFC's a bit harder to read, imo

Reply:
thinking about this more, we could swap out the `$` with the type name in place, but to truly accurately represent the 'size' we need to know both the count field (if it exists) + the subtype

Reply:
this has been updated!